### PR TITLE
Test CapacityThresholds configuration

### PR DESCRIPTION
**Pull request by mfrancisc**
Add helper functions and unit tests for new CapacityThresholds fields.

Jira Ticket: [ASC-114](https://issues.redhat.com/browse/ASC-114)

Related PRs:

- [toolchain-common](https://github.com/codeready-toolchain/toolchain-common/pull/279)

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -106,4 +106,6 @@ require (
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
+replace github.com/codeready-toolchain/toolchain-common => github.com/mfrancisc/toolchain-common v0.0.0-20221120225717-d72cdcb11147
```
Comment (MatousJobanek) [inline go.mod:109]: friendly reminder :-)

**Review by MatousJobanek — APPROVED** [review]
Looks great :+1:

File: go.mod (hunk 1 above)
Comment (mfrancisc) [reply go.mod]: fixed in [574bb7da30a68](https://github.com/codeready-toolchain/host-operator/commit/574bb7da30a684bf1c407130cc2d1f0b9315c5fb)

**Pull request merged**